src/storage/src/monitor/hummock_state_store_metrics.rs

```diff
@@ -51,6 +53,18 @@ pub struct HummockStateStoreMetrics {
     pub merge_imm_task_counts: GenericCounterVec<AtomicU64>,
     // merge imm ops
     pub merge_imm_batch_memory_sz: GenericCounterVec<AtomicU64>,
+
+    // spill task counts from unsealed
+    pub spill_task_counts_from_unsealed: GenericCounter<AtomicU64>,
+    // spill task size from unsealed
+    pub spill_task_size_from_unsealed: GenericCounter<AtomicU64>,
+    // spill task counts from sealed
+    pub spill_task_counts_from_sealed: GenericCounter<AtomicU64>,
+    // spill task size from sealed
+    pub spill_task_size_from_sealed: GenericCounter<AtomicU64>,
+
+    // uploading task
+    pub uploader_uploading_task_size: GenericGauge<AtomicU64>,
```

Comment:
Why is `uploading_task_size` a `Gauge` but `spill_task_size` is a `Counter`?

Reply:
The `uploading_task_size` is used to monitor the currently ongoing uploading task size and so we use a `Gauge`, while the `spill_task_size` is used to monitor the issuing rate of uploading task and so we the `Counter`, which is similar to monitoring the merging task size.